tests/test_connection.py

```diff
@@ -52,10 +52,11 @@
             mandatory=True,
         )
 
-    message = e.value.message
```

Comment:
That is breaking change. If you want to does this you have to support keep compatibility for public API. For example, add `@property` `message` for this exception.

IMHO this change unnecessary for this semantic context. DeliveryError returning a message and the message it's only one and this might be called just `message`.

Cost of support in existing code greater than profit.

Reply:
Child exception will make this rename unnecessary

Reply:
Actually, `message` attribute should represent repr template string according to https://github.com/mosquito/aiormq/blob/master/aiormq/exceptions.py#L6